autogen/external/spirv.core.grammar.json

```diff
@@ -132,7 +132,7 @@
         { "kind" : "IdResult" },
         { "kind" : "IdRef",                                     "name" : "'Set'" },
         { "kind" : "LiteralExtInstInteger",                     "name" : "'Instruction'" },
-        { "kind" : "IdRef",                 "quantifier" : "*", "name" : "Operands" }
+        { "kind" : "IdRef",                 "quantifier" : "*", "name" : "'Operand 1', +\n'Operand 2', +\n..." }
```

Comment:
is this really the name of an operand? We have some code that tries to make decisions based on that field.

Reply:
That's what's in upstream, so I suggest we patch up that code to make it easier to merge in the future.

Reply:
If it's a massive complication I can ping  Khronos internally and see if we can change it back.

Reply:
I think we can get around by checking the `Op` itself instead of the argument name

Reply:
Yes. That field is used to generate the spec. So I'm not sure we can convince upstream to change it.